_playwright-tests/UI/ExportSystem.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from 'test-utils';
+import { navigateToSystems } from './helpers/navHelpersPatch';
+
+test('EXPORT', async ({ page }) => {
+  await navigateToSystems(page);
+
+  await page.getByRole('button', { name: 'Export' }).click();
+  await page.getByRole('menuitem', { name: 'Export to CSV' }).click();
+
+  const preparingAlert = page.locator('div.pf-v5-c-alert.pf-m-info.notification-item');
```

Comment:
we try to only use locators that may change (like ones with patternfly in the name) as a last resort. is there another way to target the notification by using the text maybe?